bot/code_review_bot/cli.py

```diff
@@ -173,8 +181,30 @@ def main():
             extras["error_message"] = str(e)
         w.index(revision, state="error", **extras)
 
-        # Update Harbormaster status
-        w.update_status(revision, state=BuildState.Fail)
+        # Update Phabricator
+        failure = UnitResult(
+            namespace="code-review",
+            name="general",
+            result=UnitResultState.Broken,
+            details="WARNING: A generic error occurred in the code review bot.",
+            format="remarkup",
+            duration=0,
+        )
+
+        w.phabricator.update_build_target(
+            revision.build_target_phid, BuildState.Fail, unit=[failure]
+        )
+
+        # Also update lando
+        if lando_api is not None:
+            try:
```

Comment:
Let's put it behind the same config as we have in events, so we can disable it if it gets too noisy for develoeprs

Reply:
Sure!